changelogs/unreleased/fix-optional-field-no-default.yml

```diff
@@ -0,0 +1,4 @@
+---
+description: Fixes an issue about optional fields without default value
+change-type: patch
+destination-branches: [master, iso5]
```

Comment:
This issue is present in the release, so we need to add a bugfix section for it.